fiboa_cli/convert_utils.py

```diff
@@ -36,6 +36,9 @@ def convert(
         log("Loading file")
     path = download_file(url, cache_file)
 
+    if not len(bbox) == 4:
+        raise ValueError("Bbox must be of length 4")
+
```

Comment:
```suggestion
```

I've directly added this to the main branch, good idea.

Reply:
Thanks, I forgot it a few times and this error makes it clear from the start.